test/system/preTestScript.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#The script is only for jarvis environments and jarvis client should be installed.
```

Comment:
Separate this comment from the license header, please. Put at least one new line in between.

Reply:
done.